mysql-test/suite/rocksdb/t/use_direct_reads_writes.test

```diff
@@ -44,4 +53,47 @@ shutdown_server 10;
 --source include/wait_until_connected_again.inc
 --disable_reconnect
 
---exec grep "enable both use_direct_io_for_flush_and_compaction" $MYSQLTEST_VARDIR/log/mysqld.1.err | cut -d] -f2
+--let $assert_text= Checking for use_direct_io_for_flush_and_compaction
+--let $assert_file= $MYSQLTEST_VARDIR/log/mysqld.1.err
+--let $assert_only_after= use_direct_reads_writes.test restart direct_writes
+--let $assert_select= enable both use_direct_io_for_flush_and_compaction
+--let $assert_count= 1
+--source include/assert_grep.inc
+
+# Verify invalid direct-writes and --rocksdb_flush_log_at_trx_commit combination at startup fails
+--exec echo "wait" >$_expect_file_name
+shutdown_server 10;
```

Comment:
Here you can use a simple restart_mysqld.inc with a custom error log path due to the next comment, but in general, consider using shutdown_mysqld.inc instead of exec echo wait / shutdown_server where possible: both for shorter testcase and also this removes hardcoded shutdown timeouts (Percona Server with 10 second shutdown timeouts gets an occasional kill in our CI farm, which the testcase might not be prepared for)